### DISCUSSION TITLE
Does open-webui support Azure OpenAI and Azure Dall-E ?

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#2063] [General]
Does open-webui support Azure OpenAI and Azure Dall-E ?  if not, do you think is complex to add this features ?

**Comment by rodrigojuarez** [comment]
I'm interested in this one too!
Did you find anything @zouzeTG ?
At least it should support the chat part, look at the Azure section here https://github.com/open-webui/open-webui/discussions/1038

**Comment by gelbzucht** [comment]
porting and using my existing azure config for chat now for a separate litellm instance in docker worked pretty well. check: https://github.com/open-webui/open-webui/discussions/1038#discussioncomment-9665854
